executor/executor.cc

```diff
@@ -80,6 +83,10 @@ bool flag_enable_fault_injection;
 
 bool flag_collect_cover;
 bool flag_dedup_cover;
+
+// if true then executor should write the comparisons data to fuzzer
```

Comment:
Stick with the existing comment style. Full line comments are full sentences -- start with a capital letter and end with a dot.